.gitignore

```
@@ -1,6 +1,9 @@
 data/prefect_db
 data/tiled_db
 data/tiled_storage/writable
+data/results
+data/tiled_storage/recons/rec20240425*
```

This is fairly specific, perhaps we relax it to:

```suggestion
data/tiled_storage/recons
```